plugins/woocommerce/patterns/coming-soon-entire-site.php

```diff
@@ -10,7 +10,7 @@
 ?>
 
 <!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"20px","bottom":"20px"}},"color":{"background":"#bea0f2"}},"layout":{"type":"constrained"}} -->
```

Comment:
The padding you've removed needs to be also removed from the comment here. The Site Editor will throw an error when the attributes don't match.

<img width="1150" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/d1759051-03da-4c98-80f6-3ba7974bfab9">

Go to Appearance > Editor > Templates > Page: Coming soon to see the error.

Reply:
If you attempt recovery and "Copy" the block, it will likely reconcile the difference for you

<img width="459" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/7339eed0-5eaf-4d3e-bfa8-37d19abd5c0a">

Reply:
Thanks @psealock! I've fixed it in 591199e31559c73d1a3a7576276e93a87b53b014